### DISCUSSION TITLE
Next patch release v1.9.3

### DISCUSSION THREAD

**Discussion by Patrick-Remy** [CodeDredd/pinia-orm#1927] [General]
I am really interested in this fix, which is already over a month in main https://github.com/CodeDredd/pinia-orm/pull/1909 

Is it possible to release a new patch version that includes it?

Thank you so much for your work and effort to this project!

**Comment by CodeDredd** [comment]
@Patrick-Remy Going to make an release next week. Going to schedule it. Wanna put some more work into the release, after i had vacation. Surly some ppl already noticed that i was again a bit passive 😉
